drivers/hmis/app/models/hmis/hud/client.rb

```diff
@@ -216,6 +216,10 @@ def build_primary_custom_client_name
     )
   end
 
+  def masked_name
+    "Client <#{id}>"
+  end
```

Comment:
As noted above, I wonder if this should actually be initials....

Reply:
Or maybe configurable by site to either be initials _or_ client id as you have now.  I can see situations where you wouldn't want to expose anything, not sure if initials would be sufficient in those cases.

Reply:
@gigxz what do you think about this? I'm happy to implement it either way. It seems like if the primary use case, for now, is GR devs poking around prod to investigate issues while minimizing exposure, then having the masked name just point to the client ID is enough. But if this ever gets used by non-GR users, then maybe it's more friendly to have a masked name that shows the initials -- it's a bit aesthetically similar to masking all but last 4 digits of ssn.

Also, what would be the best way to implement this as a per-site configurable attribute? A new env var?

Reply:
Hm lets stick with `Client id` for now. I think this should just be `"Client #{id}"`.

I could see initials being useful in some case, but initials are still PII so I think we should err on the side of privacy. The `id` is still useful when communicating, because users can search for it and see it on the client profile. We should favor communicating with IDs over communicating with initials.


`GrdaWarehouse::Config` class in the warehouse is used to manage "site settings" which can be configured on the warehouse admin page. SO far, we have avoided using it for HMIS, but we probably will need site settings some point. I think that will probably want to be a different table. Def out of scope here though.

Reply:
Sounds like a plan.  I hadn't realized that this was all collected into one method when I started that thought, seems like it would be super easy to adjust in the future if we need to.